private.metrics.tf

```diff
@@ -1,9 +1,15 @@
+resource "random_id" "metrics" {
```

Comment:
please can we give the resources in this terraform file some more descriptive names